.eslintrc.cjs

```diff
@@ -33,7 +33,7 @@ module.exports = {
       }
     },
     {
-      files: ['types/*.ts'],
+      files: ['types/*.ts', 'test/**/*.ts'],
```

Comment:
This is unrelated, but I noticed that the the TS tests weren't being linted since they moved directories.

Reply:
Nice find. We probably want to move that to a separate PR though?

Reply:
It's a tiny enough change that I for one don't mind throwing it in here but of course if Jos objects...

Reply:
Mostly added it in this PR because the lack of linting annoyed me 😄 I went ahead and opened a separate PR for this change just to (possibly) get it in sooner. Should have done that last week 🤷